### PR TITLE
add oscp 4.18 osci configs for slorg-dotnet-core

### PR DESCRIPTION
**Pull request by amp-rh**
add oscp 4.18 osci configs for slorg-dotnet-core:
  - dotnet-interop-aws
  - dotnet-interop-aws-fips

### CODE REVIEW

**Comment by amp-rh** [comment]
/pj-rehearse pull-ci-sclorg-ansible-tests-master-dotnet-ocp4.18-lp-interop-images

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.18-lp-interop-dotnet-interop-aws periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.18-lp-interop-dotnet-interop-aws-fips

File: ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        {
+          "failure_rules":
+            [
+              {"step": "dotnet-deploy-and-test", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "dotnet-deploy-and-test", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "DOTNET", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp","self-managed-lp","dotnet-lp"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
```
Comment (dfrazzette) [inline ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.18-lp-interop.yaml:44]:
```suggestion
       FIREWATCH_DEFAULT_JIRA_ASSIGNEE: aslice@redhat.com
      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
```

File: ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.18-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        {
+          "failure_rules":
+            [
+              {"step": "dotnet-deploy-and-test", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "dotnet-deploy-and-test", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "DOTNET", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp","self-managed-lp","fips","dotnet-lp"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
```
Comment (dfrazzette) [inline ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.18-lp-interop.yaml:69]:
```suggestion
      FIREWATCH_DEFAULT_JIRA_ASSIGNEE: aslice@redhat.com
      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
```

**Pull request merged**